HandsOn/Group16/mappings/yarrrml_mapping.yml

```diff
@@ -0,0 +1,40 @@
+prefixes:
+  ns: "http://www.semanticweb.org/group16/ontologies/air-quality#"
+  rdfs: "http://www.w3.org/2000/01/rdf-schema#"
+  sc: "http://www.schema.org#"
+
+mappings:
+    
+    TriplesMap1:
+        sources:
+            - [csv-updated.csv~csv, "$1"]
+            - [estaciones.csv~csv, "$2"]
+        s:
```

Comment:
this should be
s: http://www.semanticweb.org/group16/ontologies/air-quality/$1(PUNTO_MUESTREO)